README.md

```diff
@@ -1,3 +1,295 @@
+
+1. Use the following command to find out the URL of the Gardener dashboard.
+
+    ```bash
+    sow url
+    ```
+
+More information: [Most Important Commands and Directories](#most-important-commands-and-directories)
+
+## Configuration File acre.yaml
+
+This file will be evaluated using `spiff`, a dynamic templating language for yaml files. For example, this simplifies the specification of field values that are used multiple times in the yaml file. For more information, see the [spiff repository](https://github.com/mandelsoft/spiff/blob/master/README.md).
+
+<pre>
+landscape:
+  <a href="#landscapename">name</a>: &lt;Identifier&gt;                       # general Gardener landscape identifier, for example, `my-gardener`
+
+  <a href="#landscapecluster">cluster</a>:                                          # Information about your base cluster
+    kubeconfig: &lt;relative path + filename&gt;          # Path to your `kubeconfig` file, rel. to folder `landscape`
+    domain: &lt;prefix&gt;.&lt;cluster domain&gt;               # Unique basis domain for DNS entries
+    iaas: &lt;gcp|aws|azure&gt;                           # iaas provider (coming soon: openstack|alicloud)
+    region: &lt;major region&gt;-&lt;minor region&gt;           # Example (gcp, aws): europe-west1; example (Azure): westeurope
+
+  <a href="#landscapenetworks">networks</a>:                                         # <a target="_blank" rel="noopener noreferrer" href="https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing">CIDR IP ranges</a> of base cluster
+    nodes: &lt;CIDR IP range&gt;                                  
+    pods: &lt;CIDR IP range&gt;                                   
+    services: &lt;CIDR IP range&gt;                             
+
+  <a href="#landscapeiaas">iaas</a>:
+    region: &lt;major region&gt;-&lt;minor region&gt;      # region that Gardener will use for seed clusters
+    zones:                                     # Remove zones block for providers other than GCP or AWS
+      - &lt;major region&gt;-&lt;minor region&gt;-&lt;zone&gt;   # Example: europe-west1-b
+      - &lt;major region&gt;-&lt;minor region&gt;-&lt;zone&gt;   # Example: europe-west1-c     
+      - &lt;major region&gt;-&lt;minor region&gt;-&lt;zone&gt;   # Example: europe-west1-d
+    credentials:                               # credentials to get access to the seed cluster through service account
+
+  <a href="#landscapeetcd">etcd</a>:
+    backup:
+      type: &lt;gcs|s3&gt;                          # type of blob storage
+      resourceGroup:                          # Azure specific, see below
```

Comment:
if you have a see below here, you should also hyperlink the `resourceGroup` key

Reply:
I thought it would be sufficient to have the key to the `etcd` key.

Anyway, I will change the comment to:
`# Azure resource group you would like to use for your backup`

Then it's consistent with the description in the `etcd` block I link to and users do not have to follow a link just for key `resourceGroup` - it would be difficult to link to that entry in my table.